spec/order_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'order'
+
+describe Order do
+  
+  # let(:message) {:order.confirm}
+  
+  describe '#add' do
+    it { is_expected.to respond_to(:add) }
+
+    it 'increases the order_list by 1' do
+      expect { subject.add("Kim Chi") }.to change { subject.order_list.count }.by(1) 
+    end 
+    # build a test which increases the 
+    # order_list by 1 after add
+  end 
+
```

Comment:
Do you also need a test to check that Order's add(item) method is adding the hash key value pair to the @order_list array ? e.g. something like expect (subject.order_list).to include {"kim chi" => 4}